script.js

```diff
@@ -1,4 +1,161 @@
+
+//Create a version of the Beat That dice game, where players create the largest number 
+//they can by concatenating random dice roll digits
+//There are 2 players and players take turns.
+//When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+//The player picks the order of the dice they want. 
+//For example, if they wanted the number 63, they would specify that the 2nd dice goes first. 
+//You can choose how the player specifies dice order.
+//After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+var GAME_MODE_DICE_ROLL = 'dice roll';
+var GAME_MODE_DICE_ORDER = 'dice order';
+var currentGameMode = GAME_MODE_DICE_ROLL; //initialise game mode to explain rules.
+var currentPlayer = 1;
+var diceRollOne;
+var diceRollTwo;
+var player1Score;
+var player2Score;
+
+
+
+// var player1DiceRolls = [];//keep track of dice rolls for each player
+// var player2DiceRolls = [];
+
+// var player1CombNum; //keep track of each player's combined numbers
+// var player2CombNum;
+// var gameCount = 0;//keep track of number of games played
+// var winCountPlayerOne = 0;//keep track of number of wins per user
+// var winCountPlayerTwo = 0
+
+
+
+
+var diceRolls = function (){
+  var generateDiceRolls = [diceRoll(),diceRoll()];
+
+  if (currentPlayer == 1){
+    player1DiceRolls = generateDiceRolls;
+  }
+  if (currentPlayer == 2){
+    player2DiceRolls = generateDiceRolls;
+  }
+return generateDiceRolls;
+}
+
+var playerCombNum = function (diceRoll1, diceRoll2){
+return Number (String(diceRoll1)+ String(diceRoll2));
+};
+
+// ================== Main function =====================
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = '';
+
+
+// ================== Roll dice logic =====================
+var diceRoll = function (){
+  var randomDecimal = Math.random() * 6;
+  randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+}
```

Comment:
Please put your diceRoll function outside your main function if possible since it is a helper function.